test/Libraries/PackageManagerTests/PackageLoaderTests.cs

```diff
@@ -545,7 +533,25 @@ public void MixedPackageLoadTest()
 
             // Assert value of loaded CN is non-null.
             AssertNonNull("576f11ed5837460d80f2e354d853de68");
+        }
+
+        [Test]
+        public void LoadingAPackageWithBinariesDoesNotAffectCustomNodesUsedInHomeWorkspace()
+        {
+            // Open a custom node definition and a workspace where this custom node is used.
+            OpenModel(Path.Combine(TestDirectory, @"core\PackageLoadReset\test.dyf"));
+            OpenModel(Path.Combine(TestDirectory, @"core\PackageLoadReset\MissingNode.dyn"));
+
+            // Load a package which contains binaries, when the graph is open in the workspace. 
+            var loader = GetPackageLoader();
+            var pkg = loader.ScanPackageDirectory(Path.Combine(PackagesDirectory, "Mixed Package"));
+            loader.LoadPackages(new List<Package> { pkg });
 
+            // Custom node in workspace should not have any warnings on them.
+            var functionNodes = CurrentDynamoModel.CurrentWorkspace.Nodes.OfType<Function>();
+            Assert.AreEqual(1, functionNodes.Count());
+            var functionNode = functionNodes.First();
+            Assert.AreEqual(ElementState.Active, functionNode.State);
```

Comment:
can we also assert this custom node returns the correct value?

Reply:
To be honest I'm not exactly sure what the example is doing, as I took it straight from @aparajit-pratap repro. I can probably check that it returns some value at least.

Reply:
This example load the dyf first then followed by the dyn home workspace, at this point, we should have a working custom node and a dummy node. Then we load the package and make sure custom node still function correctly. I would encourage before and after the package load, we assert custom node in Active State and give correct value.